frontend/src/pages/notebookController/screens/server/NotebookServerDetails.tsx

```diff
@@ -108,7 +108,7 @@ const NotebookServerDetails: React.FC = () => {
         <DescriptionListGroup>
           <DescriptionListTerm>Accelerator</DescriptionListTerm>
           <DescriptionListDescription>
-            {accelerator.accelerator?.spec.displayName || 'unknown'}
+            {accelerator.accelerator?.spec.displayName || 'None'}
```

Comment:
Same here -- if the value is 0 -- none makes sense... if the value > 0 -- this is odd.

Reply:
we should have an `unknown` and a `none`
so three cases
`accelerator` is defined: use `displayName`
`accelerator` is `undefined` AND `useExisting` is `true`: use `"Unknown"`
`accelerator` is `undefined` AND `useExisting` is `false`: use `"None"`